### DISCUSSION TITLE
Values of custom variables during evolution.

### DISCUSSION THREAD

**Discussion by Atulsingh92** [anyoptimization/pymoo#244] [Q&A] [answered]
Hello Julian, 

I have a method that gives a variable (vt) for each of my design points, however, as it goes indirectly to my objective function, I am unable to see it for each design point. 

Is there a way I could save , and then see the result of a variable that is not "F", "X", or "G", after the run completes?

basically I would like to get

`res.history[99].pop.get("vt")`

or perhaps `res.vt`

**Comment by blankjul** [comment, marked as answer]
everything you set to `out` in `_evlauate` will be there.
